src/components/Quotes.js

```diff
@@ -0,0 +1,45 @@
+import './Quotes.css';
+import { useEffect, useState } from 'react';
+
+function Quotes() {
+  const [quote, setQuote] = useState({});
+
+  const API_KEY = 'cnTlCQ/IFnLNbxvOCXF7fQ==9kiKfPTsk03LKemN';
+  useEffect(() => {
+    fetch('https://api.api-ninjas.com/v1/quotes?category=happiness', {
+      headers: { 'X-Api-Key': API_KEY },
+      method: 'GET',
+      contentType: 'application/json',
+    }).then((response) => response.json()).then((result) => {
+      if (result.error) {
+        setQuote({ quote: 'An Error Occurred' });
+      } else {
+        setQuote(result[0]);
+      }
+    });
+  }, []);
```

Comment:
You did a great job working on this milestone! 🚀 However, I have a few suggestions to make your code even better!

- [ ] First of all, according to the requirements:

![image](https://user-images.githubusercontent.com/77069011/227550143-c4c9fee3-bc5b-48fc-be23-e2c9149a8c5c.png)

You've handled this requirement using the same quote state, it might work but ideally, you would like your state to be responsible for just one thing (Following the Single responsibility principle). What you can do is create separate states for error and loading, like this:

```js
const [quote, setQuote] = useState(undefined);
const [loading, setLoading] = useState(true);
const [error, setError] = useState(false);
```

- [ ] Please consider using async/await to handle your asynchronous code, but be careful using it directly in your useEffect hook since this can lead to linter errors and not work as expected, you can read [this article](https://devtrium.com/posts/async-functions-useeffect) for reference. Example:

```js
  useEffect(() => {
    const fetchQuotes = async () => {
      try {
        const response = await fetch(
          'https://api.api-ninjas.com/v1/quotes?category=happiness',
          {
            headers: { 'X-Api-Key': API_KEY },
            method: 'GET',
            contentType: 'application/json',
          },
        );

        const data = (await response.json())[0];

        setQuote(data);
      } catch (error) {
        setError(true);
      }
      setLoading(false);
    };

    fetchQuotes();
  }, []);

```

Please consider applying this! 🤓 🔧